instrumentation/methods/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/methods/MethodInstrumenters.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.methods;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanKindExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanNameExtractor;
+import java.lang.reflect.Method;
+
+public final class MethodInstrumenters {
+  private static final String INSTRUMENTATION_NAME =
+      "io.opentelemetry.javaagent.external-annotations";
+
+  private static final Instrumenter<Method, Void> INSTRUMENTER;
+
+  static {
+    SpanNameExtractor<Method> spanName = new MethodSpanNameExtractor();
```

Comment:
You don't need a separate class, you can use a method reference:
```suggestion
    SpanNameExtractor<Method> spanName = SpanNames::spanNameForMethod;
```